vagrant/boxes.d/99-local.yaml.example

```diff
@@ -15,50 +33,50 @@ centos8-katello-devel:
         - "--foreman-proxy-content-enable-ostree=true"
         - "--katello-devel-modulestream-nodejs=12"
 
-centos8-luna-demo:
-  box: centos8-stream
+almalinux8-luna-demo:
+  box: almalinux8
   memory: 10240
   ansible:
     playbook: playbooks/luna_demo_environment.yml
 
-centos8-hammer-devel:
-  box: centos8-stream
+almalinux8-hammer-devel:
+  box: almalinux8
   memory: 512
   ansible:
     playbook: 'playbooks/hammer_devel.yml'
     group: 'hammer-devel'
     variables: # defaults in roles/hammer_devel/defaults/main.yml
-      #hammer_devel_server: centos8-katello-devel.custom-domain.example.com
+      #hammer_devel_server: almalinux8-katello-devel.custom-domain.example.com
 
 katello-remote-execution:
-  box: centos8-katello-nightly
+  box: almalinux8-katello-nightly
   installer: --enable-foreman-remote-execution
 
-centos8-proxy-devel:
-    box: centos8-stream
+almalinux8-proxy-devel:
+    box: almalinux8
     ansible:
       playbook: 'playbooks/foreman_proxy_content_dev.yml'
       group: 'foreman-proxy-content'
-      server: 'centos8-katello-devel'
+      server: 'almalinux8-katello-devel'
 
 centos7-katello-client:
   box: centos7
   ansible:
     playbook: 'playbooks/katello_client.yml'
     group: 'client'
     variables:
-      katello_client_server: 'centos8-katello-devel'
+      katello_client_server: 'almalinux8-katello-devel'
 
-centos8-katello-client:
-  box: centos8-stream
+almalinux8-katello-client:
+  box: almalinux8
   ansible:
     playbook: 'playbooks/katello_client.yml'
     group: 'client'
     variables:
-      katello_client_server: 'centos8-katello-devel'
+      katello_client_server: 'almalinux8-katello-devel'
 
-centos8-katello-bats-ci:
-  box: centos8-stream
+almalinux8-katello-bats-ci:
```

Comment:
Related, but not blocking for this PR: below is the CentOS 7 bats box that can't work anymore.